.factory/factory_lib.py

```
@@ -936,9 +951,10 @@ def pct(xs, q):
         for s in evidence_suites(argv[2:]):
             print(s)
         return 0
-    if cmd == "final-gate":
-        # final-gate —— 确定性测试门命令（ADR-009 数据化；链脚本 read -ra 拆词执行）
-        print(final_gate_cmd())
+    if cmd == "docstring-gate":
+        # docstring-gate —— docstring 门命令（可选键；空输出=未启用，链脚本跳过）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)